src/App.scss

```diff
@@ -1,5 +1,5 @@
-@import "./stylesheets/custom.scss";
 @import "~bootstrap/scss/bootstrap";
+@import "./stylesheets/custom.scss";
```

Comment:
Move this upside; this is contained overwrite bootstrap variable

Reply:
No, it doesn't work if we change the order, please check by running it on your side

Reply:
Created new file instead, hope that works